packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -355,8 +355,8 @@ class GtkKeyHelper with KeyHelper {
     const int controlRightKeyCode = 0xffe4;
     const int capsLockKeyCode = 0xffe5;
     const int shiftLockKeyCode = 0xffe6;
-    const int metaLeftKeyCode = 0xffe7;
-    const int metaRightKeyCode = 0xffe8;
+    const int metaLeftKeyCode = 0xffeb;
+    const int metaRightKeyCode = 0xffec;
```

Comment:
This list is in numerical order, so these should be moved below `altRightKeyCode`.

Reply:
@robert-ancell Done. Is the failing `flutter-build` something I should be worried about? (Seems to be failing for other pull requests as well)

Reply:
The failing flutter-build is the previous merge has caused a test failure. We just have to wait for a fix for that before this can be merged.